kalite/distributed/management/commands/restorebackup.py

```diff
@@ -18,9 +18,12 @@ class Command(BaseCommand):
         )
 
     def handle(self, *args, **options):
+        backup_filepath = options['backup_filepath']
         if(options['backup_filepath']):
             if(os.path.isfile(backup_filepath)):
                 call_command('dbrestore', filepath=backup_filepath, database='default') #perform restore from the given filepath
+            else:
+                print 'The given file was not found.'
```

Comment:
Maybe add a `sys.exit(1)` after this print statement, to make sure the command exits.